recipes/intel-compilers-repack/meta.yaml

```diff
@@ -0,0 +1,363 @@
+{% set version = "2022.0.1" %}          # [linux]
+{% set version = "2022.0.0" %}          # [osx or win]
+{% set intel_build_number = "3633" %}   # [linux]
+{% set intel_build_number = "3615" %}   # [osx]
+{% set intel_build_number = "3663" %}   # [win]
+
+{% set oneccl_version = "2021.5.0" %}
+{% set oneccl_build_number = "478" %}
+
+{% set tbb_version = "2021.0.0" %}
+
+# use this if our build script changes and we need to increment beyond intel's version
+{% set dst_build_number = '0' %}
+{% set build_number = intel_build_number|int + dst_build_number|int %}
+
+package:
+  name: intel-compiler-repack
+  version: {{ version }}
+
+source:
+  # Is there a better way to specify skipping the source if it does not exist? (so it matches 'skip: True' selectors)
+  - url: https://anaconda.org/intel/intel-cmplr-lic-rt/{{ version }}/download/{{ target_platform }}/intel-cmplr-lic-rt-{{ version }}-intel_{{ intel_build_number }}.tar.bz2
+    folder: intel-cmplr-lic-rt
+  - url: https://anaconda.org/intel/intel-fortran-rt/{{ version }}/download/{{ target_platform }}/intel-fortran-rt-{{ version }}-intel_{{ intel_build_number }}.tar.bz2
+    folder: intel-fortran-rt
+  {% if not win32 %}
```

Comment:
Is there a specific reason you use Jinja instead of `# [not win]`?

Also, we don't support win32 on conda-forge. Why does this appear here then?

Reply:
Main reason was consistency with anaconda repack recipe for compiler and reduce conflicts. In long term this might help with maintenance.

Reply:
I've changed from jinja to selectors. Reason for win32 is as @napetrov says.